### PR TITLE
sparkwind

### PR DESCRIPTION
**Pull request by BALAJI141031**
files to check:
1.landing route
src > routes > landing route(index.js,index.css) (**NO need to check this** )
2.components > createTweet and  components> Tweet and  components > navbar
3.routes > auth > index.js,index.css

### CODE REVIEW

File: src/components/LandingRouteHero/index.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.mainhero-text{
+    font-size: 1.2rem;
+}
+
+.hero-text{
+    font-size: 0.7rem;
```
Comment (deekshithmd) [inline src/components/LandingRouteHero/index.css:6]: Try to use px here, orelse font size will decrease to very small value while changing screen size

File: src/routes/LandingRoute/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          <NavLink to={PATHS.LOGIN}>
+            <div className="account-info">
+              <p>Already Have an Account</p>
+              <MdOutlineKeyboardArrowRight className="icon" />
+            </div>
+          </NavLink>
+        </div>
+      </div>
+    </main>
+  );
```
Comment (deekshithmd) [inline src/routes/LandingRoute/index.js:35]: code looks good, you can proceed to merge

**Review by deekshithmd — APPROVED** [review]

File: src/backend/controllers/PostController.js [hunk 3]
```diff
@@ -63,8 +64,48 @@ export const getAllUserPostsHandler = function (schema, request) {
...
+//       {},
+//       {
+//         error,
+//       }
+//     );
+//   }
+// };
 export const createPostHandler = function (schema, request) {
   const user = requiresAuth.call(this, request);
+  console.log(user, "coming here");
```
Comment (ShrutiLakhani) [inline src/backend/controllers/PostController.js:108]: Please remove all unwanted console.logs

File: src/backend/controllers/UserController.js [hunk 4]
```diff
@@ -21,7 +22,10 @@ export const getAllUsersHandler = function () {
 
 export const getUserHandler = function (schema, request) {
   const userId = request.params.userId;
+  console.log(userId, "from get up handler");
```
Comment (ShrutiLakhani) [inline src/backend/controllers/UserController.js:25]: From here as well

File: src/components/CreateTweet/index.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+.cancel-cta-div{
+    /* border-bottom: 1px solid; */
+    box-shadow: 0 2px 0px rgb(0 0 0 / 0.2);
+    margin-bottom: 5px;
+}
+
+.crate-tweet-icon{
+    height:1.5rem;
+    width:1.5rem;
+    margin-right: 15px;
```
Comment (ShrutiLakhani) [inline src/components/CreateTweet/index.css:33]: Suggestion: Avoid using PX above 10px

File: src/components/CreateTweet/index.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    width:2rem;
+    height:2rem
+}
+
+textarea:focus, input:focus{
+    outline: none;
+    border-style: solid;
+    border-width:4px;
+    border-radius:2px;
+    border-color: #B8DFD8;
```
Comment (ShrutiLakhani) [inline src/components/CreateTweet/index.css]: Suggestion: You may want to use a color variable

File: src/components/CreateTweet/index.css [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+    top:-155%;
+    padding:5px;
+    z-index: 99;
+    border: 0px solid ;
+    box-shadow: rgba(14, 30, 37, 0.12) 0px 2px 4px 0px, rgba(14, 30, 37, 0.32) 0px 2px 16px 0px;;
+    }
+
+li {
+    list-style: none;
+    margin-right:15px;
```
Comment (ShrutiLakhani) [inline src/components/CreateTweet/index.css:99]: Same as above. Please use rem or em

File: src/components/CreateTweet/index.js [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+                              type: REDUCER_CONSTANTS.TWEETTEXT,
+                              payload: tweet.tweetText + emoji,
+                            });
+                          }}
+                        >
+                          {emoji}
+                        </li>
+                      ))}
+                    </div>
+                  )} */}
```
Comment (ShrutiLakhani) [inline src/components/CreateTweet/index.js:148]: Remove the commented code if you don't wish to use it

File: src/components/Cta/index.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
+.primary-cta{
+    background-color: #FF004D;
+    color: #DBD8E3;
```
Comment (ShrutiLakhani) [inline src/components/Cta/index.css]: Please use color variables

File: src/contexts/homeContext.js [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+  console.log();
+  useEffect(() => {
+    (async () => {
+      try {
+        const postsResponse = await getAllPosts();
+        //   setHome({
+        //     type: "updatePosts",
+        //     payload: ,
+        //   });
+        // tweetsFeed = postsResponse.data.posts.reverse();
```
Comment (ShrutiLakhani) [inline src/contexts/homeContext.js:45]: Remove the commented code

**Review by ShrutiLakhani — APPROVED** [review]
Your code looks clean and well structured. Added few suggestions for CSS, you may want to check. Please remove commented code and unwanted console logs.

File: src/components/LandingRouteHero/index.css (hunk 1 above)
Comment (BALAJI141031) [reply src/components/LandingRouteHero/index.css:6]: thanks for comment but i'm not changing root font size so now its not require if i adds more responsiveness i will try to check this

**Pull request merged**